src/pint/models/timing_model.py

```diff
@@ -2062,9 +2063,12 @@ def as_parfile(
             Categories to include at the beginning
         last_order : list
             Categories to include at the end
+        comment : str, optional
+            Additional comment string to include in parfile
         """
         self.validate()
-        result_begin = ""
+        info_string = pint.utils.info_string(prefix_string="# ", comment=comment)
+        result_begin = info_string + "\n"
```

Comment:
It might be nice to be able to turn off the comments - I often use `as_parfile` as a human-readable format for parfiles while debugging, where that might not be so useful.

Reply:
good idea.  done